packages/@datapull/mysql-origin/index.js

```diff
@@ -79,6 +79,17 @@ class MySQLOrigin {
           return reject(err);
```

Comment:
Don't we want to close connection on error as well?

Reply:
nice catched! added that one as well